debian/control

```diff
@@ -0,0 +1,36 @@
+Source: python-securesystemslib
+Section: devel
+Priority: optional
+Maintainer: NYU Secure Systems Lab <securesystemslab@googlegroups.com>
+Uploaders:
+ Santiago Torres-Arias <santiago@nyu.edu>,
+ Holger Levsen <holger@debian.org>,
+ Lukas Puehringer <lukpueh@nyu.edu>,
+ Justin Cappos <jcappos@nyu.edu>,
+Build-Depends:
+ python,
+ python-nacl,
+ python-cryptography,
+ dh-python (>= 2.20160818~),
+ dpkg-dev (>= 1.17.14),
+Standards-Version: 4.1.1
+Homepage: https://ssl.engineering.nyu.edu
+Vcs-Git: https://github.com/secure-systems-lab/securesystemslib.git
+Vcs-Browser: https://github.com/secure-systems-lab/securesystemslib/
+
+Package: python-securesystemslib
+Architecture: all
+Depends:
+ dh-python,
+ python-cryptography,
+ python-nacl,
+Description: A secure cryptography and general purpose library for python.
+
+Package: python3-securesystemslib
+Architecture: all
+Depends:
+ dh-python3,
+ python3-cryptography,
+ python3-nacl,
+Description: A secure cryptography and general purpose library for python
+ (python 3 version).
```

Comment:
Are you required to have separate python2 and python3 packages of the library?  The same source code supports both Python 2 and 3.
I'm just verifying...

Reply:
This creates two separate packages for py2 and py3. Debian (and other distros) do this because the interpreters aren't shipped together.